mypy/stubgen.py

```diff
@@ -1568,6 +1571,14 @@ def mypy_options(stubgen_options: Options) -> MypyOptions:
     options.python_version = stubgen_options.pyversion
     options.show_traceback = True
     options.transform_source = remove_misplaced_type_comments
+    if stubgen_options.mypy_config_file:
+
+        def set_strict_flags() -> None:  # not needed yet
+            return
```

Comment:
I am not sure that this is correct. It would treat `strict` differently.

Reply:
I'm not too familiar with this callback so I had just replicated https://github.com/python/mypy/pull/9203
We can instead replicate [main.py](https://github.com/python/mypy/blob/master/mypy/main.py#L1216-L1221) and do
```python
    def set_strict_flags() -> None:
        for dest, value in strict_flag_assignments:
            setattr(options, dest, value)
```
would that work?

Reply:
Yeah, something like this would work (I hope so, at least)!

Reply:
I'm afraid I'm not familiar enough to know how to continue. The blurb I linked refers to a `strict_flag_assignments` variable that gets populated through some [deeper method](https://github.com/python/mypy/blob/master/mypy/main.py#L462-L495). Do you think strict flags are something that would be useful when parsing config files in `stubgen.py`? (I'm not sure of what strict flags even are, maybe with some extra context I can help implement something more robust)

Otherwise, would you be ok leaving the current implementation as is (it at least matches the existing [already merged] `stubtest.py` PR that inspired my PR) and someone more knowledgeable could propose a better handling of strict flags in the future?

Thanks for your quick response!

Reply:
Ok, if this is hard to do, we can skip it for now :)